frontend/RaffleBay/RaffleBay/Controller/ProfileViewController.swift

```diff
@@ -28,6 +28,7 @@ func get_user_request(auth_token: String, user: User) -> Void {
                 user.firstName = data["first_name"].string!
                 user.lastName = data["last_name"].string!
                 user.pic_url = data["pic_url"].string!
+                user.account_balance = String(data["balance"].int!)
```

Comment:
Why are our models storing balance information as a string? And wouldn't balance need to be unwrapped to floating-point precision not int?

Reply:
ur right about the floating point. there was a reason i kept them all as strings. i was getting an error somewhere but i forgot where.